rfc/0075-faster-failover-and-configuration-push.md

```diff
@@ -0,0 +1,373 @@
+# Meta
+
+| Field          | Value                                  |
+|----------------|----------------------------------------|
+| RFC Name       | Faster Failover and Configuration Push |
+| RFC ID         | 75                                     |
+| Start Date     | 2023-06-14                             |
+| Owner          | Sergey Avseyev                         |
+| Current Status | DRAFT                                  |
+| Revision       | #1                                     |
+
+# Summary
+
+TBD
+
+# Motivation
+
+TBD
+
+# Relation to Other RFCs
+
+This RFC relates to the following documents:
+
+* [RFC-0005][rfc-0005]: VBucket Retry Logic.
+
+* [RFC-0024][rfc-0024]: Fast-Failover SDK.
+
+
+# High-Level Design
+
+TBD
+
+# User-Facing API
+
+TBD
+
+# Implementation Details
+
+## Protocol Changes
+
+[https://issues.couchbase.com/browse/MB-57311]: #
+
+### Get Cluster Config with Known Version
+
+[https://review.couchbase.org/c/kv_engine/+/192301]: #
+
+The KV engine introduces a new HELLO flag called `GetClusterConfigWithKnownVersion` with a value of `0x1d`. This flag
+does not change the behavior of the server but allows determining if the node supports epoch-revision fields for the
+`GetClusterConfig` (`0xb5`) operation. If the node acknowledges `GetClusterConfigWithKnownVersion`, then the SDK can use
+the new version of the command.
+
+Epoch and revision are signed 64-bit integers encoded in network (big-endian) order and should be encoded as extras.
+
+
+      Byte/     0       |       1       |       2       |       3       |
+         /              |               |               |               |
+        |0 1 2 3 4 5 6 7|0 1 2 3 4 5 6 7|0 1 2 3 4 5 6 7|0 1 2 3 4 5 6 7|
+        +---------------+---------------+---------------+---------------+
+       0| 0x80          | 0xb5          | 0x00          | 0x00          |
+        +---------------+---------------+---------------+---------------+
+       4| 0x00          | 0x00          | 0x00          | 0x00          |
+        +---------------+---------------+---------------+---------------+
+       8| 0x00          | 0x00          | 0x00          | 0x00          |
```

Comment:
Is the byte in position 11 supposed to be 0x10? (Total body (8-11) : 0x00000010 (16 bytes))